src/common/components/core/TextInput.spec.tsx

```diff
@@ -65,31 +65,31 @@ describe('TextIpnut component', () => {
 
   test('TextArea has value', async () => {
     // act
-    render(<TextArea onChange={onChange} value={value} label={label} inset={true} small={true} disabled={false} />);
+    render(<TextArea onChange={onChange} value={value} label={label} inset small disabled={false} />);
 
     // assert
     expect(screen.getByRole('textbox')).toHaveValue(value);
   });
 
   test('TextInput has value', async () => {
     // act
-    render(<TextInput onChange={onChange} value={value} label={label} inset={true} small={true} disabled={false} />);
+    render(<TextInput onChange={onChange} value={value} label={label} inset small disabled={false} />);
 
     // assert
     expect(screen.getByRole('textbox')).toHaveValue(value);
   });
 
   test('TextArea has label', async () => {
     // act
-    render(<TextArea onChange={onChange} value={value} label={label} inset={true} small={true} disabled={false} />);
+    render(<TextArea onChange={onChange} value={value} label={label} inset small disabled={false} />);
 
     // assert
     expect(screen.getByText(label)).toBeInTheDocument();
   });
 
   test('TextInput has label', async () => {
     // act
-    render(<TextInput onChange={onChange} value={value} label={label} inset={true} small={true} disabled={false} />);
+    render(<TextInput onChange={onChange} value={value} label={label} inset small disabled={false} />);
```

Comment:
disabled is false by default.

Reply:
This will actually require further changes and feels out of scope. `npm run test:coverage' fails with the suggested change.